src/main/scala/rps/model/Game.scala

```diff
@@ -0,0 +1,29 @@
+package rps.model
+
+import rps.model.Move._
+import rps.model.Result._
+import rps.model.Round._
+import scala.io.StdIn.readLine
+import io.buildo.enumero.{CaseEnumSerialization}
+
+object Game {
+  val randomGen = scala.util.Random
+
+  def getRandomMove() : Move = {
+    Array(Rock, Paper, Scissors).apply(randomGen.nextInt(3))
```

Comment:
you can leave `apply` out and just do 

```scala
Array(Rock, Paper, Scissors)(randomGen.nextInt(3))
```

Reply:
Sorry, this one didn't work. It works if I split it in ```val arr = Array(...)```
and ```arr(random....)```so I keep it like this.

    [error] /Users/paismx/slask/rps-mig/src/main/scala/rps/model/Game.scala:11:51: type mismatch;
    [error]  found   : Int
    [error]  required: scala.reflect.ClassTag[rps.model.Move]
    [error]     Array(Rock, Paper, Scissors)(randomGen.nextInt(3))
    [error]                                                   ^
    [error] /Users/paismx/slask/rps-mig/src/main/scala/rps/model/Game.scala:11:10: type mismatch;
    [error]  found   : Array[rps.model.Move]
    [error]  required: rps.model.Move

Reply:
I tried also with some extra parenthesis, with no luck.

Reply:
ahh ok, yes, it's the parser that gets confused because the Array constructor takes an implicit ClassTag. Leave it as is.